scripts/SANS/SANSadd2.py

```diff
@@ -70,11 +70,11 @@ def add_runs(runs, # noqa: C901
             lastPath, lastFile, logFile, num_periods, isFirstDataSetEvent = _loadWS(
                 userEntry, defType, inst, ADD_FILES_SUM_TEMPORARY, rawTypes, period)
 
-            is_not_allowed_instrument = inst.upper() != 'SANS2D' and inst.upper() != 'LARMOR'
+            is_not_allowed_instrument = inst.upper() != 'SANS2D' and inst.upper() != 'LARMOR' and inst.upper() != 'ZOOM'
```

Comment:
This'll get cumbersome if you keep adding more instruments - could you change it to something like
```
is_allowed_instrument = inst.upper() in {"SANS2D", "LARMOR", ZOOM"}
if not is_allowed_instrument... blah
```